docs/source/55_api.md

```diff
@@ -373,6 +371,10 @@ all are called in a similar way:
   * Inner join and group by right
   * Right outer join and group by right
 
+A subset of these joins are depicted in Figure 2 below.
+
+![Joins Available](source/img/join_examples.png)
```

Comment:
Is this link accurate? I found that I had to use the relative path from the doc.